package.json

```diff
@@ -32,6 +32,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-feather": "^2.0.10",
+    "react-icons": "^4.12.0",
```

Comment:
It would be better if you use the added react-feather icon library since it is light weight. The size of react icon is [over 70 MB](https://www.npmjs.com/package/react-icons).

Reply:
replaced react-icons from react-feather